ui/component/claimSupportButton/view.jsx

```diff
@@ -12,32 +12,41 @@ type Props = {
   isRepost?: boolean,
   doOpenModal: (id: string, {}) => void,
   preferredCurrency: string,
+  doTipAccountCheckForUri: (uri: string) => void,
+  canReceiveFiatTips: ?boolean,
+  doTipAccountCheckForUri: (uri: string) => void,
+  canReceiveFiatTips: ?boolean,
 };
 
 export default function ClaimSupportButton(props: Props) {
-  const { uri, fileAction, isRepost, disableSupport, doOpenModal, preferredCurrency } = props;
+  const {
+    uri,
+    fileAction,
+    isRepost,
+    disableSupport,
+    doOpenModal,
+    preferredCurrency,
+    canReceiveFiatTips,
+    doTipAccountCheckForUri,
+  } = props;
 
-  if (disableSupport) return null;
+  React.useEffect(() => {
+    if (canReceiveFiatTips === undefined) {
+      doTipAccountCheckForUri(uri);
+    }
+  }, [canReceiveFiatTips, doTipAccountCheckForUri, uri]);
```

Comment:
:small_blue_diamond: Sometimes (usually by refreshing while in a channel page), you might get double calls if there happens to be another render before the first call is completed.  Probably need one of those local isFetching states.

<img width="490" alt="image" src="https://user-images.githubusercontent.com/64950861/185553121-b39358c9-5919-475e-91a3-3dbf23605e5f.png">